### DISCUSSION TITLE
I can't make EarTrumpet stay.

### DISCUSSION THREAD

**Discussion by Githubber616** [File-New-Project/EarTrumpet#1541] [Q&A]
I can't make EarTrumpet stay in the active "notification icons" bar. On every reboot, it is again under more icons and I have to drag it back.

My intention was that EarTrumpet replace the native system icon. I turned that native one off, and dragged EarTrumpet "in its place" but EarTrumpet only stays there till the next reboot.

What do I do wrong?

One Image says more than a thousand stuttered and mumbled words:
![image](https://github.com/File-New-Project/EarTrumpet/assets/63751987/9de62408-d598-4f63-bf52-8c65d59273ae)

**Comment by riverar** [comment]
Does EarTrumpet show up in Windows Settings > Personalization > Taskbar? Also, what's the OS version here? (Start > Run > winver)

**Reply by Githubber616** [reply]
It does. And currently:
![image](https://github.com/File-New-Project/EarTrumpet/assets/63751987/79fcb0ad-7543-4b42-9313-3eff79f99be5)
But it buggers off on next reboot. Well, I eat a broom if it doesn't, because it always did it previously.

Version: 19043.1766 21H1, and no, I will not update. I'd rather not get forced-ads for an OS my system isn't compatible with. Or any of the other shit MS pulls. Never touch a running system.
